lib/content-services/src/lib/search/components/search-filter-chips/search-facet-chip-tabbed/search-facet-chip-tabbed.component.html

```diff
@@ -0,0 +1,60 @@
+<mat-chip [attr.data-automation-id]="'search-filter-chip-tabbed-' + tabbedFacet.label"
+          disableRipple
+          class="adf-search-filter-chip-tabbed"
+          [class.adf-search-toggle-chip]="(displayValue$ | async) || menuTrigger.menuOpen"
+          [disabled]="!isPopulated()"
+          tabIndex="0"
+          [matMenuTriggerFor]="menu"
+          (onMenuOpen)="onMenuOpen()"
+          (keydown.enter)="onEnterKeydown()"
+          (keydown.escape)="onEscKeydown()"
+          [attr.title]="displayValue$ | async"
+          #menuTrigger="matMenuTrigger">
+
+    <span class="adf-search-filter-placeholder">
+        <span class="adf-search-filter-ellipsis">{{ tabbedFacet.label | translate }}:</span>
+    </span>
+
+    <span class="adf-search-filter-ellipsis adf-filter-value" *ngIf="displayValue$ | async as displayValue; else showAny">
+        &nbsp; {{ displayValue | translate }}
+    </span>
+    <ng-template #showAny><span class="adf-search-filter-ellipsis adf-filter-value">&nbsp;{{ 'SEARCH.FILTER.ANY' | translate }}</span></ng-template>
+    <mat-icon *ngIf="isPopulated(); else disabledIcon">{{ chipIcon }}</mat-icon>
```

Comment:
here as well - can we avoid call function?

Reply:
Fixed